generators/entity-client/templates/ionic/e2e/entities/_entity.po.ts.ejs

```diff
@@ -0,0 +1,179 @@
+<%#
+ Copyright 2019-Present the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+import { element, by<% if (!readOnly) { %>, ElementFinder<% } %> } from 'protractor';
+<%_
+let elementGetter = `getText()`;
+
+_%>
+export class <%= entityClass %>ComponentsPage {
```

Comment:
I just noticed you're using 4 spaces everywhere. I changed to use 2 spaces in https://github.com/oktadeveloper/generator-jhipster-ionic/pull/284.